quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/MethodDatabase.java

```diff
@@ -554,7 +555,8 @@ public boolean equals(Object obj) {
                 return false;
             }
             final ClassEntry other = (ClassEntry) obj;
-            return superName.equals(other.superName) && methods.equals(other.methods);
+            // CORDA-3756 names can be null.
+            return Objects.equals(superName, other.superName) && methods.equals(other.methods);
```

Comment:
OK, so this resolves the NPE. What does a `MethodDatabase` entry without a `superName` mean for Quasar though?